src/components/util/PersonLookup.vue

```diff
@@ -14,53 +14,28 @@
     </div>
 
     <div :class="{ 'col col-6': props.inline, 'pt-1': !props.inline }">
-      <v-autocomplete
-        :id="props.id"
-        ref="autocomplete"
-        v-model="selected"
-        v-model:search-input="search"
-        :allow-overflow="false"
-        :append-icon="null"
-        :aria-disabled="props.disabled"
-        :aria-label="toLabel(selected)"
-        auto-select-first
-        background-color="white"
-        class="person-lookup"
-        :class="props.inputClass"
-        dense
-        :disabled="props.disabled"
-        :error="!!props.errorMessage"
-        :error-messages="props.errorMessage ? [props.errorMessage] : []"
-        hide-details
-        :hide-no-data="isSearching || !search"
-        :items="suggestions"
-        light
-        :loading="isSearching ? 'tertiary' : false"
-        :menu-props="menuProps"
-        no-data-text="No results found."
-        no-filter
-        outlined
-        :placeholder="props.placeholder"
-        return-object
-        single-line
-        @blur="onBlur"
-        @focus="onFocus"
-        @update:list-index="onHighlight"
-      >
-        <template #selection="{ item }">
-          <span class="text-nowrap">{{ toLabel(item) }}</span>
-        </template>
-        <template #item="{ item, attrs, on }">
-          <v-list-item
-            v-bind="attrs"
-            :aria-selected="item === highlightedItem"
-            class="tertiary--text"
-            v-on="on"
-          >
-            <span v-html="suggest(item)" />
-          </v-list-item>
-        </template>
-      </v-autocomplete>
+      <AccessibleCombobox
+        id-prefix="search-options-note-student"
+        aria-description="Student name or S I D lookup. Expect auto suggest."
```

Comment:
@mohamalnadi - I believe we want an `aria-description` that describes instructor attributes, yes?